### PR TITLE
Build projects in parallel

### LINKED ISSUE #1216: Have MSBuild build in parallel
Any reason not to?

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #1216.

### CODE REVIEW

File: build.csx [hunk 1]
```diff
@@ -245,7 +245,7 @@ public void RunMsBuild(string target)
     var packagesDirectoryOption = string.IsNullOrEmpty(packagesDirectory) ? "" : $"/p:NuGetPackagesDirectory={packagesDirectory}";
     Cmd(
         msBuild,
-        $"{solution} /target:{target} /p:configuration=Release /nr:false /verbosity:minimal /nologo /bl:artifacts/logs/{target}.binlog {packagesDirectoryOption}");
+        $"{solution} /target:{target} /p:configuration=Release /maxcpucount /nr:false /verbosity:minimal /nologo /bl:artifacts/logs/{target}.binlog {packagesDirectoryOption}");
```
Comment (thomaslevesque) [inline build.csx]: Unrelated to your change, but do you have any idea why we have `/nr:false`? Was node reuse causing issues at some point? Disabling node reuse is probably not great for performance.

Comment (thomaslevesque) [reply build.csx]: Just tried with `/nr:true`; the difference isn't significant (1 or 2 second difference for a total time of ~40s)

**Comment by thomaslevesque** [comment]
Thanks @blairconrad! It does seem to improve the build time.

File: build.csx (hunk 1 above)
Comment (blairconrad) [reply build.csx]: I have no idea why we have `/nr:false`, and no opinion about it being on or off. Whatever you like.

**Pull request merged**